bundles/ranvier-commands/commands/get.js

```diff
@@ -63,6 +63,7 @@ module.exports = (srcPath) => {
       player.addItem(item);
 
       Broadcast.sayAt(player, `<green>You receive loot: </green>${item.display}<green>.</green>`);
+      Broadcast.sayAt(room, `<green>${player.name} receives loot: </green>${item.display}<green>.</green>`)
```

Comment:
Don't like this, there's no reason anyone else should know I just picked up an item and I could be abusive to other players by constantly dropping my inventory and picking it up.